### PR TITLE
Add reusable components

### PR DESCRIPTION
**Pull request by clmnshn28**
- Created a 'components' folder to organize reusable components.

### CODE REVIEW

File: frontend/src/components/NewUserTextField.js
Comment (otaKenneth) [inline frontend/src/components/NewUserTextField.js:1]: Why NewUserTextField, you can just name it TextField.

File: frontend/src/components/PasswordRequirements.js
Comment (otaKenneth) [inline frontend/src/components/PasswordRequirements.js:1]: You should put this with PasswordField component.

File: frontend/src/pages/admin/account/AccountSettingsAdmin.js [hunk 1]
```diff
@@ -4,12 +4,32 @@ import { Link } from 'react-router-dom';
...
+      { label: 'Street Address', value: 'Everlasting St.' },
+      { label: 'Barangay', value: 'Bulihan' },
+      { label: 'City', value: 'Malolos' }
+    ]
+  ];
+
   return (
 
-      <div className="bgaccount-container">
+      <MainContent>
```
Comment (otaKenneth) [inline frontend/src/pages/admin/account/AccountSettingsAdmin.js:32]: What happens to <slot/> component used in Routes?

**Review by otaKenneth — CHANGES_REQUESTED** [review]
Maximize the used of Routes/Slots components.

File: frontend/src/components/NewUserTextField.js
Comment (clmnshn28) [reply frontend/src/components/NewUserTextField.js:1]: okay po sir

**Review by otaKenneth — APPROVED** [review]

**Pull request merged**